src/fairseq2/models/w2vbert/builder.py

```diff
@@ -43,6 +43,7 @@ def _encoder_600m() -> Wav2Vec2EncoderConfig:
         layer_drop_p=0.0,
         norm_order=TransformerNormOrder.POST,
         depthwise_conv_kernel_size=31,
+        conv_norm_type="batch_norm",
```

Comment:
Since "batch_norm" is the default anyways, I don't think that this extra argument pass is needed.